sentry_irc/plugin.py

```diff
@@ -94,17 +94,18 @@ def get_group_url(self, group):
     def notify_users(self, group, event, fail_silently=False, **kwargs):
         link = self.get_group_url(group)
         message = event.message.replace('\n', ' ').replace('\r', ' ')
-        if event.server_name:
```

Comment:
not following, what was this resolved to ?

Reply:
event.server_name was deprecated and removed (it previously returned `self.get_tag("server_name")` though). This is the only place where it was used (and according to @scefali this plugin doesn't actually work)

Reply:
Correct, this plugin does not work currently

Reply:
Although this "fix" just results in a different error, I'll still merge this so we can get rid of the legacy event usage consistently throughout sentry

Reply:
Well it might work. It’s possible I don’t have IRC configured correctly which causes it to time out. Once it’s on prod, I can test it with other installations.

Reply:
https://sentry.io/organizations/sentry/issues/1358609972/?environment=staging&project=1&referrer=alert_email

It works, it’s misconfigured.